native/src/utils/openExternalUrl.ts

```diff
@@ -27,6 +28,10 @@ const openExternalUrl = async (rawUrl: string, showSnackbar: (snackbar: Snackbar
         },
       })
       InAppBrowser.close()
+      // On ios InAppBrowser seems to need some time to close the browser until it can be opened again properly
```

Comment:
Nice, could you perhaps link the issue here? 
```suggestion
      // On ios InAppBrowser seems to need some time to close the browser until it can be opened again properly
      // https://github.com/digitalfabrik/integreat-app/issues/3084
```